src/imitation/scripts/__init__.py

```diff
@@ -9,5 +9,3 @@
     # pytype: enable=import-error
 except ImportError:
     pass
-
-import imitation.envs.examples  # noqa: F401
```

Comment:
Sorry, I ran the tests and it they all still passed. There is a bug in sphinx that does not let do autosummary build when you leave that line. We can roll back using autosummary, or we can try to see if a relative import fixes it. (see https://github.com/sphinx-doc/sphinx/issues/9069).

Reply:
Yeah, we should probably be testing this in `test_scripts.py`, e.g. could do an `eval_policy` on `imitation/Random-v0`, although it's not a core functionality (just some demo environments being available by default).

I'm not sure what to do about the bug. Open since Apr 2021 so I guess not getting closed soon, which is pretty sad.

I don't want to block this closing the PR so I'll just open https://github.com/HumanCompatibleAI/imitation/issues/501 to track it and we can leave it as is.